docs/admin/admission-controllers.md

```diff
@@ -78,6 +98,11 @@ If your cluster supports containers that run with escalated privileges, and you
 restrict the ability of end-users to exec commands in those containers, we strongly encourage
 enabling this plug-in.
 
+### GenericAdmissionWebhook (alpha)
+
+This plug-in is related to the [Dynamic Admission Control](/docs/admin/extensible-admission-controllers)
```

Comment:
This plug-in calls the webhooks configured via `ExternalAdmissionHookConfiguration`, and only admits the operation if all the webhooks admit it. Currently the plugin always fails open, i.e., ignore the failed call to a webhook.

Reply:
Done.